src/assets/style.scss

```diff
@@ -286,6 +295,7 @@ table {
 
 .content-article {
   max-width: $width-content;
+  width: 100%;
```

Comment:
Ensures the article takes up the full width allocated to it by the grid system even when the content within the article is narrower (e.g. stub articles with no long text).